pkg/microservice/aslan/core/common/service/service.go

```diff
@@ -82,18 +82,22 @@ type ServiceProductMap struct {
 }
 
 // ListServiceTemplate 列出服务模板
-func ListServiceTemplate(productName string, log *zap.SugaredLogger) (*ServiceTmplResp, error) {
+func ListServiceTemplate(productName, envName string, log *zap.SugaredLogger) (*ServiceTmplResp, error) {
```

Comment:
don't pass envName, it make no sense!

Reply:
two services use one document , no better way for now

Reply:
add a router